docs/workshop/lesson2.md

```diff
@@ -129,7 +129,7 @@ Before you start creating your assessments, make sure you have at least 4 questi
 
 ### Configuration 4: Synchronous exam using PrairieTest
 
-When using [PrairieTest](https://us.prairietest.com/pt/docs/staff) to schedule and deliver PrairieLearn exams, you need to using the following configuration:
+When using [PrairieTest](https://us.prairietest.com/pt/docs/center/setup-center) to schedule and deliver PrairieLearn exams, you need to using the following configuration:
```

Comment:
This should not point to the center staff docs. Point to the course staff docs instead: https://us.prairietest.com/pt/docs/course/welcome

Reply:
Fixed

Reply:
It looks like you're still pointing to the center staff documentation?

Reply:
I am so sorry about that. I know your time is valuable and I won't make mistakes like that again. Now it has been fixed and is ready to be merged!